build_catalog.sh

```diff
@@ -0,0 +1,62 @@
+#!/bin/bash
+
+if [ -z "${BUNDLE_IMAGE_NAME}" ]; then
+  echo "BUNDLE_IMAGE_NAME is not set"
+  exit 1
+fi
+
+if [ -z "${CATALOG_IMAGE_NAME}" ]; then
+  echo "CATALOG_IMAGE_NAME is not set"
+  exit 1
+fi
+
```

Comment:
I usually do this in the following way to capture everything in one go

```bash
count=0
for var in BUNDLE_IMAGE_NAME \
           CATALOG_IMAGE_NAME \
           QUAY_USER \
           QUAY_TOKEN
do
    if [ ! "${!var}" ]; then
        echo "$var is not set"
        count=$((count + 1))
    fi
done

[ $count -gt 0 ] && exit 1
```

Reply:
I didn't include QUAY_USER/TOKEN simply because they are part of the CI/CD, but you're right they should be included.  I added ^^.